network/src/commonMain/kotlin/com/wire/kalium/network/api/conversation/ConversationApiImpl.kt

```diff
@@ -16,10 +16,12 @@ class ConversationApiImpl internal constructor(private val authenticatedNetworkC
 
     private val httpClient get() = authenticatedNetworkClient.httpClient
 
+    @Suppress("MagicNumber")
     override suspend fun fetchConversationsIds(pagingState: String?): NetworkResponse<ConversationPagingResponse> =
         wrapKaliumResponse {
             httpClient.post("$PATH_CONVERSATIONS/$PATH_LIST_IDS") {
-                setBody(PaginationRequest(pagingState = pagingState))
+                // fixme: adjust "size" value later on when server can handle the load
+                setBody(PaginationRequest(pagingState = pagingState, size = 200))
```

Comment:
nitpick:

Can we extract this `200` to a constant? Name it something more meaninful for now, like `MAX_CONVERSATION_DETAILS_COUNT`, and we can move the `// fixme:` there, explaining that this number should be higher once it's fixed in the back-end.